src/NuGet.Core/NuGet.Packaging/Signing/SignatureTrust.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+
+namespace NuGet.Packaging.Signing
+{
+    /// <summary>
+    /// Represents the trust result of a signature.
+    /// </summary>
+    public enum SignatureTrust
+    {
+        /// <summary>
+        /// Default unknown value.
+        /// </summary>
+        Unknown = 0,
+
+        /// <summary>
+        /// Invalid signature.
+        /// </summary>
+        /// <remarks>If the package has been modified after signing this will occur.</remarks>
```

Comment:
I'd think a long list of reasons could result in an invalid state.  For example, unsupported hash/signature algorithm, unmet certificate requirements, etc.  So you might just add, "For example, ..."